app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,9 @@
   <% if current_user %>
   <script>
     window.currentMenuPlan = new MenuPlan(<%= raw MenuPlanPresenter.new(current_user.current_menu_plan).to_json %>);
+    window.currentMenuPlan._recipes = new Recipes(
+      <%= raw ActiveModelSerializers::SerializableResource.new(current_user.current_menu_plan.recipes, adapter: :attributes).to_json %>
+    );
```

Comment:
What do you think of using a presenter here instead of introducing ActiveModelSerializers?